GVFS/GVFS.Platform.Mac/MacFileSystemVirtualizer.cs

```diff
@@ -380,6 +380,7 @@ private bool TryGetSymLinkTarget(string sha, out string symLinkTarget)
                             byte[] buffer = new byte[4096];
                             uint bufferIndex = 0;
                             int nextByte = stream.ReadByte();
+                            int bufferSize = 0;

```

Comment:
Naming suggestion: `bufferSize` -> `bytesWritten`

The name `bufferSize` made me think this was somehow related to `byte[] buffer`